installer/configurator/configurator.go

```diff
@@ -35,15 +38,51 @@ func validateInput(appID, site, clientToken, arch string, sessionSampleRate, ses
 	return nil
 }
 
-func handleError(err error) {
-	// TODO: Send telemetry
-
+func handleError(err error, sender *TelemetrySender, appId string, dryRun bool) {
 	log.Error(err)
 
+	if !dryRun {
+		var errorType ErrorType
+
+		if installerErr, ok := err.(*InstallerError); ok {
+			errorType = installerErr.ErrorType
+		} else {
+			errorType = UnexpectedError
+		}
+
+		if sender != nil {
+			if err := sender.SendLog(LogMessage{
+				Message: err.Error(),
+				Level:   "ERROR",
+				Tags: keyValuePairsAsTags(map[string]string{
+					"app_id":              appId,
+					"installer_version":   InstallerVersion,
+					"integration_name":    IntegrationName,
+					"integration_version": IntegrationVersion}),
+			}); err != nil {
+				log.Error("Error sending error log: ", err)
+			}
+
+			if err := sender.SendInstallationCount("error", errorType.String()); err != nil {
+				log.Error("Error sending error count: ", err)
+			}
+
+			end := time.Now().UnixNano() / int64(time.Millisecond)
```

Comment:
[question] Can `start` and `end` be in Nanoseconds and convert in millisecond while computing the delta?